### DISCUSSION TITLE
Would be nice to have 2 "aliases" for consistency.

### DISCUSSION THREAD

**Discussion by JalonSolov** [vlang/v#11663] [Ideas]
Right now, the docs list these common types:

i8    i16  int  i64      i128 (soon)
byte  u16  u32  u64      u128 (soon)

It would be nice if "u8" were an alias for "byte", and "i32" for "int".

That would give full consistency across the integer types/sizes, and leave no ambiguity as to size.

I know 'byte' and 'int' are commonly used, but they are actually different sizes on different architectures... for example, at least one old system had an 11 bit 'byte'.

**Comment by JalonSolov** [comment]
Oh, yeah... pretty much everywhere I've ever seen it, 'byte' was a signed value, but you have it in the 'u8' (unsigned 8-bit) position.  Even C makes you put 'unsigned byte' if you want unsigned.

**Comment by medvednikov** [comment]
Byte is unsigned in Go and Swift.

**Comment by medvednikov** [comment]
There's a reason i32 and u8 are skipped, I'll post a detailed article about it soon.

**Comment by RiordanIX** [comment]
Is there an update on this? I'd like to know why no aliases for these.
